conf/messages

```diff
@@ -98,6 +98,40 @@ setup.footer.copyright=&copy; 2015 yetu AG
 setup.footer.privacy=Privacy Policy
 setup.footer.imprint=Imprint
 setup.footer.terms=Terms &amp; conditions
+setup.next=next &gt;
+
+#setup registration page
+setup.registration.register=Register at yetu
+setup.registration.already=Already registered?
+setup.registration.hint=all fields are mandatory
+setup.registration.firstname=First name
+setup.registration.lastname=Last name
+setup.registration.email=E-mail
```

Comment:
some of these fields already exist in the same file. Please use existing fields where possible to avoid duplication.

```
securesocial.signup.title=Sign Up
securesocial.signup.username=Username
securesocial.signup.firstName=First Name
securesocial.signup.lastName=Last Name
securesocial.signup.email1=E-Mail
securesocial.signup.email2=E-Mail confirmation
securesocial.signup.password1=Password
securesocial.signup.password2=Password confirmation
```